### DISCUSSION TITLE
Questions about POTA Reference and Gridsquare

### DISCUSSION THREAD

**Discussion by WrathPak** [wavelog/wavelog#1252] [Q&A] [answered]
Hi! I have been using Wavelog for about a month now and am really enjoying it. I do have one pain point though, and it could be from my lack of understanding on how to do it properly. 

When I log contacts, where the person I am logging is at a POTA Park, I fill in their callsign first, then park ID in the Park Reference on the General Tab of the Live QSO. I notice once I have filled in their callsign, it pulls the gridsquare info automatically. but when i enter the POTA reference, it does not update the gridsquare to the POTA park gridsquare. I have to look it up and update it manually. 

I looked at all of the settings, and see options for pulling location when entering POTA reference ID but it doesn't seem to automatically  update the gridsquare field on the live qso page when enabled. Am I doing something wrong?

Secondly, Is there a way to have POTA Reference field on the QSO live screen by default so I don't have to move over to the General Tab to enter it? I mostly hunt POTA or activate, so I fill in that field a lot.

**Comment by HB9HIL** [comment, marked as answer]
Hello @WrathPak 

We have a little known bug there. The map is not updating but the gridsquare itself does. Just click in the gridsquare field and the next to it. It should update the map then (map data is not stored, only the grid).

Requirement for the POTA Update is the option in the user settings in the "Wavelog Preferences" tab.

<img width="560" alt="image" src="https://github.com/user-attachments/assets/08f2094b-7a27-4f2e-8ed5-5d03a152c255">

To permanently move the POTA field to the first tab you just have to scroll a little bit more down in the user settings and enable the pota switch. 

Don't forget to hit the save button ;-)

<img width="596" alt="image" src="https://github.com/user-attachments/assets/4d0d1624-0554-44b8-aecd-2d352b2242b0">

**Reply by HB9HIL** [reply]
PS: I fixed the little bug real quick in [this PR](https://github.com/wavelog/wavelog/pull/1254). Upon the next release the map updates itself aswell when entering a POTA, SOTA , WWFF

**Comment by WrathPak** [comment]
I am all sorted now. I must not have saved settings or something for the POTA auto lookup. I found the option you referenced to put the POTA Reference on the general tab too. Thanks @HB9HIL !
